crates/starknet-devnet-core/src/starknet/mod.rs

```diff
@@ -132,6 +133,18 @@ impl Starknet {
     pub fn new(config: &StarknetConfig) -> DevnetResult<Self> {
         let defaulter = StarknetDefaulter::new(config.fork_config.clone());
         let mut state = StarknetState::new(defaulter);
+
```

Comment:
Why both classes are being declared

Reply:
Users might want to predeploy cairo0 accounts, but later on be able to deploy cairo1 accounts, or vice versa.

Reply:
I added this note to the issue: https://github.com/0xSpaceShard/starknet-devnet-rs/issues/373

Reply:
okay, I get it